flang/test/Lower/OpenMP/atomic01.f90

```diff
@@ -0,0 +1,71 @@
+! RUN: bbc -fopenmp -emit-fir %s -o - | \
+! RUN: FileCheck %s --check-prefix=FIRDialect
+! RUN: bbc -fopenmp %s -o - | \
+! RUN: tco --disable-llvm --print-ir-after=fir-to-llvm-ir 2>&1 | \
+! RUN: FileCheck %s --check-prefix=LLVMIRDialect
```

Comment:
Can you also check the LLVMIR using tco considering lowering to LLVMIR has been supported?

Reply:
I checked using `bbc -fopenmp -emit-fir filename.f90 -o - | tco`. In both atomic read and write, I get unsupported OpenMP operation and LLVM translation failed. Possibly @shraiysh has not cherry-picked those to fir-dev yet. I can introduce the complete flow while upstreaming the patch. Does that work?

Reply:
Sounds good to me.